rcjaRegistration/rcjaRegistration/settings.py

```diff
@@ -321,3 +323,21 @@
 AWS_PRIVATE_MEDIA_LOCATION = ''
 
 DEFAULT_FILE_STORAGE = 'rcjaRegistration.storageBackends.PrivateMediaStorage'
+
+if not DEV_SETTINGS:
+    sentry_sdk.init(
+        dsn='SENTRY_DSN',
+        enable_tracing=False,
+        sample_rate=1.0,
+        send_default_pii=False,
+        enrionment='production',
```

Comment:
Perhaps also make this an environment variable, SENTRY_ENV?
Also should be "**environment**" not "**enrionment**"